src/test/resources/sample_docs/fever/collection1/segment1.jsonl

```diff
@@ -0,0 +1,5 @@
+{"id": "", "text": "", "lines": ""}
+{"id": "Domain_Range_Ratio_-LRB-DRR-RRB-", "text": "", "lines": ""}
+{"id": "Shohei_Otani", "text": "is a Japanese baseball pitcher , outfielder , and designated hitter who plays for the Hokkaido Nippon-Ham Fighters of Nippon Professional Baseball 's -LRB- NPB -RRB- Pacific League . He was the first pick of the Fighters in the 2012 draft . He officially recorded the fastest pitch by a Japanese pitcher and in NPB history at 165 km/h . ", "lines": "0\tis a Japanese baseball pitcher , outfielder , and designated hitter who plays for the Hokkaido Nippon-Ham Fighters of Nippon Professional Baseball 's -LRB- NPB -RRB- Pacific League .\tHokkaido Nippon-Ham Fighters\tHokkaido Nippon-Ham Fighters\tbaseball\tbaseball\tpitcher\tpitcher\tNippon Professional Baseball\tNippon Professional Baseball\tPacific League\tPacific League\n1\tHe was the first pick of the Fighters in the 2012 draft .\n2\tHe officially recorded the fastest pitch by a Japanese pitcher and in NPB history at 165 km/h .\tpitcher\tpitcher\n3\t"}
```

Comment:
Is the first line always like this?

Reply:
Nope, this was just an edge case I found in the wikipedia dumps.